remoteLib/src/SmartButton.jsx

```diff
@@ -1,10 +1,9 @@
 import React from "react";
-import { Button } from "@material-ui/core";
 
 const SmartButton = () => {
-    return <Button>
+    return <div>
```

Comment:
Why change this to a div? Should this be just plain html button?

If we are removing the @material-ui dep should that be removed from the package.json?